.github/workflows/install.yml

```diff
@@ -0,0 +1,74 @@
+name: Installation
+on:
+  push:
+    paths:
+      - "install/**"
```

Comment:
It didn't run because the PR is in draft node. The matrix context object isn't available within the script block, so I have to write these as separate jobs -- didn't get time to do it yet.